platform/ext/target/nordic_nrf/nrf9160/partition/flash_layout.h

```diff
@@ -51,7 +51,7 @@
  */
 
 /* Size of a Secure and of a Non-secure image */
-#define FLASH_S_PARTITION_SIZE                (0x30000)       /* S partition: 192 kB*/
+#define FLASH_S_PARTITION_SIZE                (0x31000)       /* S partition: 196 kB*/
```

Comment:
Why is this different from nRF5340?

Reply:
Slipup, I forgot to change this one. Thanks